crates/interpreter/src/instructions/opcode.rs

```diff
@@ -264,7 +265,7 @@ pub const OPCODE_JUMPMAP: [Option<&'static str>; 256] = [
     /* 0x47 */ Some("SELFBALANCE"),
     /* 0x48 */ Some("BASEFEE"),
     /* 0x49 */ Some("BLOBHASH"),
-    /* 0x4a */ None,
+    /* 0x4a */ Some("BLOBFEE"),
```

Comment:
does it matter that the opcode is called `BLOBBASEFEE` in the spec here?

Reply:
This: https://github.com/bluealloy/revm/pull/721#issuecomment-1721200005
I like BLOBFEE more :)

Reply:
ah I missed that, yeah I like `BLOBFEE` more too

Reply:
Renamed back to `BLOBBASEFEE` per Eth R&D discussion